portfolio/src/main/webapp/index.html

```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="getServerMessage()">
```

Comment:
Remove the `onload="getServerMessage()"` from the HTML.

Prefer to have separation of concerns for cleanliness. Page content (in HTML) should ideally be separate from style (in CSS) and behavior/code (javascript), especially as project(s) start to scale this separation becomes much more preferable.

Some alternatives include:

1. using the window.onload (Note there is some *slight* difference between the window.onload and the body.onload depending on the page).
2. Using jquery for when that part is ready.
3. Adding a normal event listener.

Reply:
changed to window.onload!
the tutorials don't have separation >.>